apps/crn-server/src/config.ts

```diff
@@ -71,3 +76,9 @@ export const clientId = SQUIDEX_CLIENT_ID || 'squidex-client-id';
 export const clientSecret = SQUIDEX_CLIENT_SECRET || 'squidex-client-secret';
 export const appName = SQUIDEX_APP_NAME || 'asap-local';
 export const sentryTraceSampleRate = 1.0;
+export const contentfulSpaceId = CONTENTFUL_SPACE_ID || 'contentful-space-id';
+export const contentfulAccessToken =
+  CONTENTFUL_ACCESS_TOKEN || 'contentful-access-token';
+export const contentfulEnvId = CONTENTFUL_ENV_ID || 'contentful-environment-id';
+export const contentfulHost = CONTENTFUL_HOST || 'https://cdn.contentful.com';
+export const isContentfulEnabled = IS_CONTENTFUL_ENABLED || false;
```

Comment:
maybe this should be evaluated into a boolean here rather than later?

Reply:
Do you mean using something like this https://github.com/node-config/node-config/issues/272#issuecomment-162567709? :)